services/identity/src/main/java/com/crapi/constant/UserMessage.java

```diff
@@ -34,6 +34,13 @@ public class UserMessage {
       "User registered successfully! Please Login.";
   public static final String SIGN_UP_FAILED = "User registered failed! Please retry.";
   public static final String NUMBER_ALREADY_REGISTERED = "Number already registered! Number: ";
+  public static final String NUMBER_NOT_REGISTERED = "Given Number is not registered! Number:";
+  public static final String CHANGE_PHONE_MESSAGE =
+      "The otp has been sent to your email. If you have used example.com email, check your email using the MailHog web portal.";
+  public static final String NUMBER_CHANGE_SUCCESSFUL = "Phone number change is successful";
+  public static final String NEW_NUMBER_DOES_NOT_BELONG = "Incorrect new number supplied";
+  public static final String OLD_NUMBER_DOES_NOT_BELONG =
+      "Old number does not match with user's number";
```

Comment:
```suggestion
      "Fail, mobile parameter doesn’t belong to the user";
```
Lets follow the same message convention as email, for example

Reply:
Sure